src/dynamic_bridge.cpp

```diff
@@ -327,6 +327,9 @@ void update_bridge(
     }
   }
 
+  int service_execution_timeout;
+  ros1_node.param<int>("service_execution_timeout", service_execution_timeout, 5);
```

Comment:
i guess it is okay to have the common parameter for all bridges. but if we want to, we can use dedicated parameter for each bridge types.

e.g) `ros1_bridge/dynamic_bridge/service_execution_timeout` and `ros1_bridge/parameter_bridge/service_execution_timeout` ?

Reply:
we could use `getParamCached` to process this faster, so that we do not query the parameter to rosmaster every single second.

Reply:
Makes sense, updated :)